src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -59,6 +59,14 @@ class CoordinateMapBase : public PUP::able {
   virtual std::unique_ptr<CoordinateMapBase<SourceFrame, TargetFrame, Dim>>
   get_clone() const = 0;
 
+  /// \brief Retrieve the same map but going from `SourceFrame` to
+  /// `Frame::Grid`.
+  ///
+  /// This functionality is needed for injecting time-dependence into domains,
+  /// where the target frame of the time-independent map is `Frame::Grid`.
```

Comment:
Is there enough functionality in place for you to give a snippet for how you expect this function to be used with the time-dependent maps?

Reply:
It's used inside the `Block.inject_time_dependence` function (see #2022) with just a call to it, i.e.
```cpp
moving_mesh_logical_to_grid_map_ = stationary_map_->get_to_grid_frame();
```
There are places that do this in the tests, e.g. 
```cpp
  const auto rotated2d_base_from_inertial =
      rotated2d_base_inertial->get_to_grid_frame();
```
What part of this would information would be helpful? Should I say that it'll be used internally by `Block` when injecting time dependence?

Reply:
I think the part I found confusing is that it's not completely clear from this context alone what 'injecting' means. From the other PR it seems the meaning is that you are creating a full map on the domain from the map composition M_2 * M_1, where M_1 was the original map and M_2 is the new time-dependent map, and that the intermediate frame after the first map and before the second map is to be labelled as `Frame::Grid`.

Reply:
Hmm, I can see that. I'm not sure this is the right place to explain moving meshes. What if I omit that part of the dox or change injecting to `adding`?

Reply:
perhaps `composing time-dependent maps with time-independent maps, where...`? I agree that the full details should probably be put in the next PR.

Reply:
Okay, done! Thanks :)